### ISSUE TITLE
DataFrame.fillna() not able to take method as a parameter

### ISSUE THREAD

**Issue by esvhd** [dask/dask#1812]
Hi guys,

Tried to use `dask.dataframe.fillna()` with `method='ffill'`, this gave the following error. 

```python
TypeError: fillna() got an unexpected keyword argument 'method'
```

According to the API docs [here](http://dask.pydata.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.fillna), `fillna()` should have been able to take a lot more parameters, just like `pandas`'s. 

Is this a function that is yet to be implemented? Would like to use `ffill` or the likes without having to provide a fill value.

Thanks.

**Comment by TomAugspurger** [comment]
`ffill` and `bfill` should be relatively easy to do now that `map_overlap` is done in https://github.com/dask/dask/pull/1769

For the documentation issue, see https://github.com/dask/dask/issues/1815

**Comment by jcrist** [comment]
Fixed by #1819.

**Comment by esvhd** [comment]
Thank you guys!
